modules/tensor_mechanics/test/include/materials/PowerLawCreepTest.h

```diff
@@ -0,0 +1,61 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "PowerLawCreepStressUpdate.h"
+#include "ADPowerLawCreepStressUpdate.h"
+
+template <bool is_ad>
+using PowerLawCreepStressUpdateTempl =
+    typename std::conditional<is_ad, ADPowerLawCreepStressUpdate, PowerLawCreepStressUpdate>::type;
```

Comment:
I have a follow on PR that will remove this, but I didn't want to pollute this here.